setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+setup(
+    name='tensorizer',
+    version='0.0.1',
+    description='A tool for PyTorch Module, Model, and Tensor Serialization/Deserialization.',
+    url='https://github.com/coreweave/tensorizer',
+    author='CoreWeave',
+    license='MIT',
+    packages=['tensorizer'],
+    install_requires=[
+        'torch',
```

Comment:
Should there be a version specification on torch? Probably applies to requirements file as well.

Reply:
Yep, the tensorizer uses the `meta` device which is only available from 1.9.0 onwards.